firefox-ios/Client/Experiments/Experiments.swift

```diff
@@ -136,7 +137,7 @@ enum Experiments {
     }
 
     /// The `NimbusApi` object. This is the entry point to do anything with the Nimbus SDK on device.
-    static var shared: NimbusInterface = {
+    nonisolated(unsafe) static var shared: NimbusInterface = {
```

Comment:
```suggestion
    // TODO FXIOS-12602 This global property is not concurrency safe
    nonisolated(unsafe) static var shared: NimbusInterface = {
```
We should probably add a ticket for this one too? I made you one here: https://mozilla-hub.atlassian.net/browse/FXIOS-12602